crates/api/src/chronicle_graphql/mod.rs

```diff
@@ -569,7 +569,7 @@ where
     M: ObjectType + 'static,
     S: SubscriptionType + 'static,
 {
-    #[instrument(level = "debug", skip(self, prepare_req), ret(Debug))]
+    #[instrument(level = "trace", skip(self, prepare_req), ret(Debug))]
```

Comment:
Might instead want to just skip the `req` for this and below, as `req` effectively includes the JWT claims.